desktop/extensions-sdk/extensions/DISTRIBUTION.md

```diff
@@ -1,54 +1,52 @@
+Extension images must be multi-arch images so that users can install extensions on ARM/AMD hardware. These multi-arch images can include ARM/AMD specific binaries. Mac users will automatically use the right image based on their architecture.
+Extensions that install binaries on the host must also provide Windows binaries in the same extension image.
 
-Docker Desktop includes an allow-list of extensions available to users. The extension image Hub repository (like `mycompany/my-desktop-extension`) must be part of the Docker Desktop allow-list to be recognized as an extension.
+> See how to [build a multi-arch image](multi-arch.md) for your extension.
 
-This allow-list specifies which Hub repositories are to be used by Docker Desktop to download and install extensions with a specific version at a given point in time.
+You can implement extensions without any constraints on the code repository. Docker doesn't need access to the code repository in order to use the extension. You can entirely manage new releases of your extension, without any dependency on Docker or Docker Desktop.
 
-Any new image pushed to a repository that is part of the allow-list corresponds to a new version of the extension. Image tags are used to identify version numbers. Extension versions must follow semver to make it easy to understand and compare versions.
+## New releases and updates
 
-With a given release of Docker Desktop (including some extensions), users should not need to upgrade Docker Desktop in order to obtain new versions of a specific extension. Newer versions of the extension can be released independently of Docker Desktop releases, provided there is no Extension API mismatchs.
+You can release a Docker Desktop extension by pushing a new image to Docker Hub with a new tag corresponding to the new release.
 
-Docker Desktop scans the list of published extensions for new versions regularly, and provides notifications to users when they can upgrade a specific extension.
+Any new image pushed to an image repository corresponding to a Docker Extension defines a new version of that extension. Image tags are used to identify version numbers. Extension versions must follow semver to make it easy to understand and compare versions.
 
-Users can download and install the newer version of an extension without updating Docker Desktop itself.
+Docker Desktop scans the list of extensions published in the marketplace for new versions, and provides notifications to users when they can upgrade a specific extension. Extensions that aren't part of the marketplace don't have automatic update notifications at the moment.
 
-## API dependencies
+Users can download and install the newer version of any extension without updating Docker Desktop itself.
 
-Extensions must specify the Extension API version they rely on. Currently there is no technical validation of this version, as the extension framework is still experimental.
+## API dependencies
```

Comment:
**suggestion**: be more explicit about what API we are talking about. I used to think it was about API in general, so I was confused.
```suggestion
## Extensions API dependencies
```